public/styles/main.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  text-transform: uppercase;
+}
```

Comment:
Don't forget about newlines
![image](https://user-images.githubusercontent.com/96409563/218975628-74b26611-0cde-43ce-8199-a6023774234a.png)
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file